device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/convention/TelemetryMessage.java

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.azure.sdk.iot.device.convention;
+
+import com.microsoft.azure.sdk.iot.device.Message;
+import lombok.Getter;
+import lombok.Setter;
+
+public class TelemetryMessage extends Message
+{
+    @Getter
+    @Setter
+    public ClientPropertyCollection Telemetry;
```

Comment:
```suggestion
    public ClientPropertyCollection telemetry;
```

fields should be lowercase